.github/workflows/dependabot-automerge.yml

```
@@ -0,0 +1,38 @@
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.user.login == 'dependabot[bot]'
+    steps:
+      - name: Dependabot metadata
+        id: metadata
+        uses: dependabot/fetch-metadata@d7267f607e9d3fb96fc2fbe83e0af444713e90b7
+        with:
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+
+      - name: Auto-approve minor, patch, and submodule updates
+        if: |
+          steps.metadata.outputs.update-type == 'version-update:semver-minor' || 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)